index.html

```diff
@@ -63,7 +64,7 @@ <h2>A project for first-time contributions</h2>
                 data-show-count="false"
                 title="Tweet this project"
               >
-                Tweet
+                <i class="fa-brands fa-twitter"></i>
```

Comment:
I can't see this as an improvement. The text is more accessible for screen readers